### PR TITLE
Resolving Add login/register feature #53 and refactor, restructure code base

### PR DESCRIPTION
**Pull request by jkubkub**
- Add login and register functionality based on SQLite database
- Add bootstrap styling
- Minor changes in design, add logo
- Restructure code base, make it more modular
- Add html templates
- Add configuration file config.py 
- Update requirements.txt

I hope it will help organize project a little bit,  can be kind a skeleton for templates, modules perspective at least

### CODE REVIEW

**Comment by jkubkub** [comment]
All app configuration is in config.py. Must mention that it is better to change secret key there.

File: templates/home.html [hunk 1]
```diff
@@ -1,81 +1,52 @@
...
-      <div class="marginBottom1">We will be working with services like: GitHub, AWS and others. Some of those services require payments. Those costs are on your own expense.</div>
-      <div>Costs</div>
-      <div class="marginBottom1">Participation is free of charge and commitment</div>
-      <div>Weekly Shared Lesson</div>
-      <div class="marginBottom1">Every Tuesday 10AM Israel Time via Skype</div>
-      <div>If you'd like to add a feature or correct a defect, please open a pull request.</div>
-      <div class="marginBottom1">If you have questions or suggestions, please open an issue.</div>
-      <div class="marginBottom1">And be sure to follow us on Facebook and LinkedIn.</div>
-      <div>And ... join our Facebook Group and our Meetup Group.</div>
+        <h3>What does than mean?</h3>
```
Comment (bregman-arie) [inline templates/home.html]: spelling

File: templates/home.html [hunk 2]
```diff
@@ -1,81 +1,52 @@
...
+        The community mission is transfering DevOps knowledge by practice. We are implementing an open source web app (www.devopsloft.io), while picking up DevOps technologies, like: git (GitHub), AWS, Linux, scripting, Jenkins, Chef/Ansible.
```
Comment (bregman-arie) [inline templates/home.html]: same

File: templates/home.html [hunk 3]
```diff
@@ -1,81 +1,52 @@
...
+        <br>
+        The goal of the community is addressing the lack of exprianced DevOps specialists to fulfil the increasing demand.
```
Comment (bregman-arie) [inline templates/home.html]: same

Comment (bregman-arie) [inline templates/home.html]: same

Comment (jkubkub) [reply templates/home.html]: What does that mean?
The community mission is transferring DevOps knowledge by practice. We are implementing an open source web app (www.devopsloft.io), while picking up DevOps technologies, like: git (GitHub), AWS, Linux, scripting, Jenkins, Chef/Ansible. 
The goal of the community is addressing the lack of experienced DevOps specialists to fulfill the increasing demand.

@bregman-arie Did you mean that?

Comment (lioramilbaum) [reply templates/home.html]: @jkubkub the work 'than' should be 'that'. I think.

Comment (jkubkub) [reply templates/home.html]: @lioramilbaum Yep , spotted that. Missed some letters also in different words. Pushed changes already.

**Comment by jkubkub** [comment]
@bregman-arie @lioramilbaum I basically totally changed these files, moved to another directory, but I cannot resolve the merge conflict because don’t have write access. Can I get it just for this PR?

**Comment by jkubkub** [comment]
@bregman-arie so what is the deal with PR?

**Comment by bregman-arie** [comment]
You don't need write access to resolve a conflict.
You need to change your PR and resubmit it. Can you do it?

**Comment by jkubkub** [comment]
@bregman-arie sure will make in a hour

**Pull request merged**